.buildkite/scripts/dockerhub_upload.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source .buildkite/scripts/shared.sh
+version=$(buildkite-agent meta-data get "version")
+
+if [[ "${FAKE_RELEASE_TAG}" ]]; then
+  export IMAGE_NAME="habitat/fakey-mc-fake-face-studio"
```

Comment:
Why not "Habby McHabFace"?

WHY NOT @elliott-davis ??

Reply:
I am made of fail

Reply:
Same question here, re: `IMAGE_NAME`